warehouse/filters.py

```diff
@@ -177,3 +177,8 @@ def contains_valid_uris(items):
 
 def parse_version(version_str):
     return packaging.version.parse(version_str)
+
+
+def uncapitalize(text):
```

Comment:
jinja has the built-in capitalize filter so I could also:

1. remove this filter
2. update [convertToReadableText](https://github.com/cheungnj/warehouse/blob/add-last-released-to-timestamp/warehouse/static/js/warehouse/utils/timeago.js#L27) to return all lowercase values
3. then invoke the capitalize filter on any existing references to humanize

Reply:
I could also use the `lower` filter. There's a lot of options...

Reply:
`lower` would be the easiest approach, yet it reads wrong when displaying a date, i.e `Released apr 19 2018` vs `Released Apr 19 2018` (ideally `Released on April 19 2018`).

I think we should separate the Jinja side from the Javascript side. If you're up for it we could:
- Extend the function `timeago.js` to read an optional suffix from a second attribute and deal with the capitalization if present.
- Remove `Released` text and the `uncapitalized` filter so it simply displays the date as before.

Reply:
Definitely up for it but if I'm reverting my changes so the date is displayed as before, then do we really need to extend `timeago.js`?

Reply:
I think prefixing the `Released` would be a nice touch, but it might not justify the change? thoughts @brainwane @nlhkabu?